src/export/ExportFFmpegDialogs.cpp

```diff
@@ -195,9 +195,13 @@ const std::vector< int > AC3BitRateValues{
 
 const int ExportFFmpegAC3Options::iAC3SampleRates[] = { 32000, 44100, 48000, 0 };
 
-ExportFFmpegAC3Options::ExportFFmpegAC3Options(wxWindow *parent, int WXUNUSED(format))
+ExportFFmpegAC3Options::ExportFFmpegAC3Options(
```

Comment:
This is beyond the scope of this PR.

All the constructors for the various ExportFFmpeg options, apart from the ExportFFmpegCustomOptions, are identical.  Could these be refactored to use an inheriting constructor, say ExportFFmpegBase, with ExportFFmpegCustomOptions' mFormat and mCodec being default initialised.  However, you will need to implement a form of the template method pattern, maybe via CRTP, to call the derived class' PopulateOrExchange.